ansible/group_vars/all

```diff
@@ -17,7 +17,7 @@ nginx_vhosts:
     server_name: "{{ hostname }} _"
     root: "{{ biblebox_default_content_root }}"
     index: "index.html index.php"
-    error_page: "404 /index.html"
+    error_page: "404 /404.html"
```

Comment:
The idea behind having the 404 page as index.html is that when someone uses an existing browser window (say http://a.b.c/somepage.html) they'll be redirected to the biblebox root page (as expected), rather than a 404-like page. What do you think is the most appropriate response?